packages/core/src/components/Menu/index.tsx

```diff
@@ -74,11 +74,13 @@ export default withStyles(({ color, ui, pattern }) => ({
 
       // These are jank. Better way?
       '> li:first-child > *': {
+        borderTop: `${unit}px solid white`,
```

Comment:
Transparent make more sense?

Reply:
![Screen Shot 2019-07-11 at 11 12 39 AM](https://user-images.githubusercontent.com/8676510/61074551-d42ebf00-a3cc-11e9-8e41-2a11ae452d48.png)

I made this change, but then realized background color shows through transparent borders, so now it's back to white, but derived properly from `color.accent.bg` so it will always match the menu background.